components/contribute-cards/ContributeCrypto.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { defineMessages, injectIntl } from 'react-intl';
+
+import { ContributionTypes } from '../../lib/constants/contribution-types';
+
+import Contribute from './Contribute';
+
+const messages = defineMessages({
+  title: {
+    id: 'Crypto',
+    defaultMessage: 'Crypto Contribution',
+  },
+  description: {
+    id: 'CollectivePage.Contribute.Crypto.Description',
+    defaultMessage: 'Make a crypto contribution through The Giving Block.',
```

Comment:
Doesn't matter that it's through the The Giving Block, remove.

Reply:
Changed to remove Giving Block.